docs/user_docs/spec_file_format.md

```diff
@@ -0,0 +1,35 @@
+# Spec file format
+**Will be rewritten** 
+This product uses CSV for file reading, this means that group definition is separated by a  ",". The spec for this product will have the following group definitions: name, ring quadrant, isNew, moved and description. For the CSV, this will look like:
+```
+name,ring,quadrant,isNew,moved,description
+```
+The reader will then translate these to structs which get translated to types using unmarshalling.
+
+The types foreach group:  
+```  
+- name        : string
+- ring        : string
+- quadrant    : string
+- isNew       : bool
+- moved       : int
+- description : string
+```
+
+Some of the groups is defined to take specific inputs. Groups can have these inputs:
+```
+- name        : The name of a data management, datastore, infrastructure or langauge.
+- ring        : The name of the ring it's place like adopt, trial, assess or hold.
+- quadrant    : The name of the quadrant like data management, datastore, infrastructure or langauge.
+- isNew       : True or false if the tech is new or not for the tech radar.
+- moved       : 0-3 depending on what ring the tech moved to. so going from hold to adopt is going from 0 to 3 is moved=3.
```

Comment:
I think the number is just a numeric representation of the ring it is moving to. So if a blip is in ring 0 and you change it to moved=3 then it would move to ring 3 and if you then wanna move it back you just change the value to 0 if that makes sense. Think this does need to be written better tho 
Alternative:
"moved : 0-3 this is a numeric representation of the 4 rings (insert names of rings here and what number e.g. adopt:2), if a blip needs to be moved you write the number that corresponds to the ring the blip should move to e.g. *insert example here*"

Reply:
Should move to, or has been moved from?

I feel like this is getting more complicated ^^"

Reply:
These a some good point, and after our discussion I would agree that moved should change. I changed it to move and updated it's description.